tests/test_infra_status.py

```diff
@@ -31,8 +32,8 @@
     "outages",
     [
         (
-            [OUTAGE_FULL, OUTAGE_NO_ENDDATE, OUTAGE_NO_ENDDATE_NO_TICKET, OUTAGE_NO_TICKET],
-            [OUTAGE_FULL, OUTAGE_NO_ENDDATE, OUTAGE_NO_ENDDATE_NO_TICKET, OUTAGE_NO_TICKET],
+            [OUTAGE_FULL, OUTAGE_NO_ENDDATE, OUTAGE_NO_TICKET, OUTAGE_NO_ENDDATE_NO_TICKET],
```

Comment:
Was there a reason to re-order things here? If it's just for style reasons I'd recommend to leave the change out and submit it as a separate PR so it's clear why the change occurred.